packages/contracts/contracts/StateChannelTransaction.sol

```diff
@@ -48,16 +47,13 @@ contract StateChannelTransaction {
       "App is not finalized yet"
     );
 
-    Transfer.Transaction memory txn = appRegistry.getResolution(
+    bytes memory resolution = appRegistry.getResolution(
       appIdentityHash
     );
 
-    require(
-      Transfer.meetsTerms(txn, terms),
-      "Transfer details do not meet terms"
-    );
-
-    txn.execute();
+    bytes memory payload = abi.encodeWithSignature("interpret(bytes,bytes)", resolution, interpreterParams);
```

Comment:
what should be included in interpreterParams? and in any specific order?

Reply:
depends on the interpreter